README.md

```diff
@@ -6,36 +6,22 @@
 
 ## Getting started
 
-1. Install node.js (get it from [nodejs.org](http://nodejs.org/)).
+* Install node.js (get it from [nodejs.org](http://nodejs.org/)).
   * If working behind a proxy, you need to configure it properly (HTTP_PROXY / HTTPS_PROXY / NO_PROXY environment variables)
-2. Install grunt-cli and bower globally
 
-   ```sh
-npm install grunt-cli bower -g
-```
-3. Clone the repository and navigate into it
-
-   ```sh
+* Clone the repository and navigate into it
+```sh
 git clone https://github.com/SAP/openui5-sample-app.git
 cd openui5-sample-app
 ```
-
-1. Install all npm dependencies
-
-   ```sh
+* Install all npm dependencies
```

Comment:
Maybe add some information about the bower dependencies here as well:
`Install all npm dependencies (also installs all bower dependencies)`